src/Controller/AbstractApiResourceController.php

```diff
@@ -140,14 +153,82 @@ public function getApiResourcesAction()
             in_array($orderByField, $this->getRepository()::getSortableFields())
             && in_array(strtolower($orderByDirection), ['asc', 'desc'])
         ) {
-            $queryBuilder->orderBy($queryBuilder->getRootAliases()[0] . '.' . $orderByField, $orderByDirection);
+            $queryBuilder->orderBy($queryBuilder->getRootAliases()[0].'.'.$orderByField, $orderByDirection);
```

Comment:
.php_cs-Fixer has concat_space = 'one'. You can run `vendor/bin/php-cs-fixer fix ./src` to prevent this mistake.